src/main/java/org/kainos/ea/cli/User.java

```diff
@@ -0,0 +1,43 @@
+package org.kainos.ea.cli;
+
+public class User {
+    private final int userId;
+    private final String email;
+    private final Role role;
+
+    public String getHashedPassword() {
+        return hashedPassword;
+    }
+
+    private String hashedPassword;
+
+
+    public User(int userId, String email, Role role, String hashedPassword) {
+        this.userId = userId;
+        this.email = email;
+        this.role = role;
+        this.hashedPassword = hashedPassword;
+    }
+
+
+
+    public User(int userId, String email, Role role) {
+        this.userId = userId;
+        this.email = email;
+        this.role = role;
+    }
```

Comment:
Tom suggested that I add it in one of the resolved comments above. I think because I was returning an Object[] in getUserByEmail so he suggested to add a constructor so i could return a User object instead